maintenance/wikia/AutomaticWikiAdoptionGatherData.php

```diff
@@ -1,32 +0,0 @@
-<?php
-/**
- * AutomaticWikiAdoptionGatherData
- *
- * An AutomaticWikiAdoption extension for MediaWiki
- * Maintenance script for gathering data - mark wikis available for adoption
- *
- * @author Maciej Błaszkowski (Marooned) <marooned at wikia-inc.com>
- * @date 2010-10-08
- * @copyright Copyright (C) 2010 Maciej Błaszkowski, Wikia Inc.
- * @license http://www.gnu.org/copyleft/gpl.html GNU General Public License 2.0 or later
- * @package MediaWiki
- * @subpackage Maintanance
- *
- */
-
-require_once( dirname(__FILE__) . '/../commandLine.inc' );
-
-if (isset($options['help'])) {
-	die( "Usage: php AutomaticWikiAdoptionGatherData.php [--quiet] [--maxwiki=12345]
```

Comment:
How to make sure that this command is no longer used?

Reply:
if it's not configured to run in chef-repo and not referred to here in app either, then it's essentially unused :)

Reply:
I couldn't find anything, so for me it looks good, but please double check it while CR